plugins/provisioners/podman/cap/linux/podman_installed.rb

```diff
@@ -0,0 +1,26 @@
+module VagrantPlugins
+  module PodmanProvisioner
+    module Cap
+      module Linux
+        module PodmanInstalled
+          def self.podman_installed(machine)
+            paths = [
+              "/bin/podman",
+              "/usr/bin/podman",
+              "/usr/local/bin/podman",
+              "/usr/sbin/podman",
+            ]
+
+            paths.each do |p|
+              if machine.communicate.test("test -f #{p}", sudo: true)
+                return true
+              end
+            end
```

Comment:
Instead of testing a list of explicit paths, a single test with `command -v podman` will validate if podman is installed and available via the configured `PATH`